ports/mercenarykings/README.md

```diff
@@ -0,0 +1,47 @@
+## Notes
+
+Thanks to:
+* [Trubute Games](https://store.steampowered.com/developer/tributegames), the developers of Flinthook.
+* JanTrueno for porting and patching Flinthook.
```

Comment:
I haven't double checked but Flinthook seems to be another game, sorry if I'm wrong :D
```suggestion
* [Trubute Games](https://store.steampowered.com/developer/tributegames), the developers of Mercenary Kings.
* JanTrueno for porting and patching Mercenary Kings.
```